lua/packer/compile.lua

```diff
@@ -33,6 +33,20 @@ vim.api.nvim_command('packadd packer.nvim')
 local no_errors, error_msg = pcall(function()
 ]]
 
+
+local enter_packer_compile = [[
+vim.g.packer_inside_compile = true
```

Comment:
Nitpick: I'd prefer that this/the other `vim.g` variables currently used be switched to the `_G.packer` table for consistency.

Reply:
Sure thing, I tried to grep for this but only found `_G.packer_plugins`, is this the one you mean? Or did you mean a new table?

Reply:
We create a `_G._packer` in e.g. line 76 of packer.lua or line 94 of compile.lua - this is meant to be the global "packer metadata" table. It's just slightly nicer than relying on variables with specific names in Vim's global scope, imo.

Reply:
Ah I see, I missed the `_` prefix of `packer`. Done in 5effb94da662e5052cd1ec3d958853efb57bcd90 :+1: